src/instructlab/lab.py

```diff
@@ -625,6 +626,42 @@ def chat(
         if not api_base:
             api_base = ctx.obj.config.serve.api_base()
 
+    # if only the chat is running (`ilab chat`) and the temp server is not, the chat interacts with
+    # in server mode (`ilab serve` is running somewhere) and thus we want to print the model used by
+    # the server
+    if not is_temp_server_running():
+        try:
+            models = list_models(
+                api_base=api_base,
+                tls_insecure=tls_insecure,
+                tls_client_cert=tls_client_cert,
+                tls_client_key=tls_client_key,
+                tls_client_passwd=tls_client_passwd,
+            )
+
+            # Currently, we only present a single model so we can safely assume that the first model
+            server_model = models.data[0].id if models is not None else None
```

Comment:
I don't think this is a safe assumption, since you can point `lab chat` at any OpenAI compatible API endpoint.

Reply:
That makes sense, I overlooked that. That complicates the fix a little. Do you have an alternative in mind?

Reply:
I would think that if the model option was provided to `ilab chat`, that has to be included in the OpenAI request (pretty sure it is, because I know it works). If that's the case, then we can always use the one from the command line arg if present.

If none is specified, we need a sane default. That could either be from config, or just default to the first one in whatever we get back from the API. I don't have a strong opinion, really.

I think in all cases, we should be displaying whatever value we're sending in the API request.

Reply:
I believe that this is what the proposed code does already, I'll add an extra line and more comments to make it clear. Thanks!